client/.eslintrc

```diff
@@ -4,10 +4,12 @@
     "google"
   ],
   "env": {
-    "browser": true
```

Comment:
this file is git pulled from master

Reply:
If it's already in master, why add it to this pull request?